main/zoe/guide/contracts/multipoolAutoswap.md

```diff
@@ -0,0 +1,169 @@
+const saraResult = await saraSeat.getOfferResult();
+```
+
+If the result is `Swap successfully completed.`, she got the simoleans for 220 or
+less (she'll want to deposit any refund). Otherwise the market price moved against
+her, and she'll have to check the price again and make another offer.
+
+```js
+const moolaRefund = await E(saraSeat).getPayout('In');
+const simoleanProceeds = await E(saraSeat).getPayout('Out');
+
+const moolaRefundAmount = E(sMoolaPurse).deposit(moolaRefund);
+E(simoleanPurse).deposit(simoleanProceeds);
+```
+
+###  Creating a new Pool
+
+When the contract is first instantiated, there won't be any pools ready for
+ trading. `addPool()` adds a new currency, which can then be funded. When a pool is
+first funded, there's no other basis on which to decide how much liquidity to create,
+so the liquidity amount equals the amount of the central token in the offer.
+
+```js
+const moolaLiquidityIssuer = await E(publicFacet).addPool(moolaIssuer, 'Moola');
+const moolaLiquidityAmountMath = await makeLocalAmountMath(moolaLiquidityIssuer);
+```
+
+Alice decides that the current rate in the external market is 2 Moola for each
+Buck, so she deposits twice as many Moola as Bucks.
+
+```js
+const aliceProposal = harden({
+  want: { Liquidity: moolaLiquidity(50) },
+  give: {
+    Secondary: moolaAmountMath.make(100),
+    Central: bucksAmountMath.make(50),
+  },
+});
+const alicePayments = {
+  Secondary: aliceMoolaPayment,
+  Central: aliceBucksPayment,
+};
+
+const aliceAddLiquidityInvitation = E(publicFacet).makeAddLiquidityInvitation();
+const addLiquiditySeat = await zoe.offer(
+  aliceAddLiquidityInvitation,
+  aliceProposal,
+  alicePayments,
+);
+```
+
+### Adding Liquidity to an Existing Pool
+
+When adding or removing liquidity to pools that have already been established, the
+amounts deposited must be in proportion to the current balances in the pool. The
+calculation is based on the amount of the `Central` asset. The `Secondary` assets
+must be added in proportion.  If less `Secondary` is provided than required, the
+offer is exited with no trade. If more of the secondary is provided than is required,
+the excess is returned.
+
+Bob calls `getPoolAllocation()` to find the relative levels. Let's say the answer is
+that the current ratio is 1234 moola to 1718 bucks.
+
+```js
+const moolaPoolAlloc = E(publicFacet).getPoolAllocation(moolaBrand);
+const bucksValue = moolaPoolAlloc.Central.value;
+const moolaValue = moolaPoolAlloc.secondary.value;
+```
+
+Then he can add liquidity. The price ratio changes when anyone trades with the pool,
+and the pool calculates the amount of `secondary` currency required based on the
+amount of `central` currency provided, so he'll bump up the amount of moola
+contributed. If he was concerned about how much liquidity this would produce, he
+could calcuate it and specify a ballpark figure, but there's usually no need to do
+that.
```

Comment:
Very long sentences with several things in them. How about
"The price ratio changes when anyone trades with the pool. So the pool calculates
the amount of `secondary` currency required based on the
amount of `central` currency provided. So he'll need to bump up the amount of his moola
contribution. If he was concerned about how much liquidity this would produce, he
could calcuate it and specify a ballpark figure. There's usually no need to do
that though.